server/test/repository/ApplicationRepositoryTest.java

```diff
@@ -144,8 +145,11 @@ private Applicant saveApplicant(String name) {
     return applicant;
   }
 
-  private Program saveProgram(String name) {
-    Program program = new Program(name, "desc", name, "desc", "", DisplayMode.PUBLIC.getValue());
+  private Program createProgram(String name) {
```

Comment:
suggestion: name createDraftProgram, some tests in our code want to create Active programs so good to be clear what state this is setting up.